### DISCUSSION TITLE
new curse api?

### DISCUSSION THREAD

**Discussion by Netspider** [Senth/minecraft-mod-manager#184] [Q&A] [answered]
Hello,
do you use the new curseforge api or the deprecated one that will stop working 2022-05-09 (in 2 days)?
https://support.curseforge.com/en/support/solutions/articles/9000208346-about-the-curseforge-api-and-how-to-apply-for-a-key

I love mmm, thanks for your work!

**Comment by Senth** [comment, marked as answer]
Hi @Netspider,

Actually, mmm uses the old API😅 Thanks for bringing this up 🙂

But yeah, my bad for not checking when they would close down the old API since I had a hunch they would do that eventually. Hopefully, I can get an API key. Otherwise, I might have to remove CurseForge from the support which kind of defeats the purpose since most mods are there.

The new Overwolf client also supports Fabric now, even though it lacks some functionality.

If I may ask, how do you use mmm? For servers, clients, or both?

**Reply by Netspider** [reply]
I am playing single player, so no server. I requested an api key yesterday and the autoreply said "You should expect an answer over the next few weeks."

**Reply by Senth** [reply]
Thanks for the answer 🙂

I've requested an API key now for mmm, but if that is rejected we always have the option that everyone needs to supply their own API keys even though that is a bit more cumbersome.
